src/update-action-hook-perms.js

```diff
@@ -0,0 +1,52 @@
+const editRole = require('./util/edit-role');
+
+module.exports.setup = (program) => {
+  return program
+    .command('update-action-hook-perms')
+    .option('-n, --noop', 'Don\'t change roles, just show difference')
+    .description('update action-related `hooks:trigger-hook:` scopes in mozilla-group:* roles');
+};
+
+EXPECTED_PERMS = [
+  {group: 'vpn_releasemgt', actions: [
+    {trustDomain: 'gecko', level: '3', actionPerm: 'relpromo'},
+  ]},
+  {group: 'releng', actions: [
+    {trustDomain: 'gecko', level: '3', actionPerm: 'relpromo'},
```

Comment:
Oh, I meant to change this from relpromo to something more innocuous like purge-cache to start with.  I don't want to mess with "scary things" yet!  But yes, this will be the place to add such scopes.